firmware/lucidgloves-firmware/AdvancedConfig.h

```diff
@@ -1,16 +1,23 @@
 //Advanced settings, only for the pros XD
 
-#define LOOP_TIME 4 //How much time between data sends (ms), set to 0 for a good time :)
+#define LOOP_TIME 1 //How much time between data sends (ms), set to 0 for a good time :)
```

Comment:
Please keeps this default as it was, or open up a new PR on why you think it should be changed.

Reply:
I'm afraid I have currently neither the time nor the knowledge of the opengloves driver to add native support for this. I might look into it sometime in the future but until then my reroute mod should work